Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -17,15 +19,31 @@ public class LootLockerConfig : ScriptableObject
 
         public static LootLockerConfig Get()
         {
+
             if (settingsInstance != null)
             {
                 return settingsInstance;
             }
+            //Try to load it
             settingsInstance = Resources.Load<LootLockerConfig>("Config/LootLockerConfig");
+
+#if UNITY_EDITOR
```

Comment:
Since this is a possibility (not finding the settingsinstance) for editor and we have this code path. Should we not do a critical exception, or at the very least log, if we're not in the editor and there's no LootLockerConfig?

Reply:
It would be very uncommon for it to happen, but sure it can happen. You would need to create a script that uses a LootLocker function, and then never test it in the Editor, just instantly make a playable build without testing it first.

Could maybe do this a bit better actually, we could create it before it's being referenced. That would solve this, since it would create the config-file instantly.
So If you have installed LootLocker, the Config-file will be created directly after installing.

This code will still be here, but I'll make an additional commit to make sure that the config-file always exists.